Solutions/Microsoft Defender Threat Intelligence/Playbooks/MDTI-Data-Cookies/AzureDeploy.json

```diff
@@ -0,0 +1,630 @@
+        "title": "MDTI-Data-Cookies",
+        "description": "This playbook uses the MDTI Components data to automatically enrich incidents generated by Microsoft Sentinel. Leverage this playbook in order to enrich your incidents with [Cookies](https://learn.microsoft.com/en-us/defender/threat-intelligence/data-sets#cookies) data hosted by the indicators found within the incident. These values sometimes contain a state for the application or little bits of tracking data. Defender TI highlights and indexes cookie names observed when crawling a website and allows users to dig into everywhere we have observed specific cookie names across its crawling and data collection.",
+        "prerequisites": [
+            "This playbook inherits API connections created and established within a base playbook. Ensure you have deployed [MDTI-Base](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Microsoft%20Defender%20Threat%20Intelligence/Playbooks/MDTI-Base/azuredeploy.json) prior to deploying this playbook. If you have trouble accessing your account or your credentials contact your account representative (mdtidiscussion[@]microsoft.com)."
+        ],
+        "lastUpdateTime": "2023-04-24T00:00:00.000Z",
+        "entities": [],
+        "tags": [],
+        "postDeployment": [
+            "After deploying the playbook, you must authorize the connections leveraged.",
+            "1. Visit the playbook resource.",
+            "2. Under 'Development Tools' (located on the left), click 'API Connections'.",
+            "3. Ensure each connection has been authorized.",
+            "**Note: If you've deployed the [MDTI-Base](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Microsoft%20Defender%20Threat%20Intelligence/Playbooks/MDTI-Base/azuredeploy.json) playbook, you will only need to authorize the Microsoft Sentinel connection.**"
+        ],
+        "support": {
+            "tier": "microsoft"
+        },
+        "author": {
+            "name": "Yaniv Shasha, Microsoft"
+        },
+        "releaseNotes": [{
+                "version": "1.0.1",
+                "title": "MDTI Data Cookies",
+                "notes": [
+                    "Updated version with Secure Inputs for HTTP REST and Secure Output for MDTI-Base actions"
+                ]
+            }
+        ]
+    },
+    "parameters": {
+        "PlaybookName": {
+            "defaultValue": "MDTI-Data-Cookies",
+            "type": "String"
+        }
+    },
+    "variables": {
+        "AzureSentinelConnectionName": "[concat('azuresentinel-', parameters('PlaybookName'))]"
+    },
+    "resources": [{
+            "type": "Microsoft.Web/connections",
+            "apiVersion": "2016-06-01",
+            "name": "[variables('AzureSentinelConnectionName')]",
+            "location": "[resourceGroup().location]",
+            "properties": {
+                "api": {
```

Comment:
Properties of connection should have other details as well , like .

"displayName": "",
"customParameterValues": {},
 "parameterValueType": "Alternative",